model/riscv_insts_end.sail

```diff
@@ -27,6 +27,8 @@ mapping clause assembly = C_ILLEGAL(s) <-> "c.illegal" ^ spc() ^ hex_bits_16(s)
 /* ****************************************************************** */
 
 /* End definitions */
+end extension_supported
```

Comment:
Stale

Reply:
I've gone back and forth on whether it's worth having an explicit end for scattered definitions. In this case it does have the use of telling the pattern completeness checker that the enumeration is now complete.

Reply:
It does strike me as odd that the Sail compiler doesn't error out on an end for something that it's never seen before, though

Reply:
Oh if that's whats happening here that's just a bug I think